### PR TITLE
Include refactoring steps + scope data into develop branche

### PR DESCRIPTION
**Pull request by drelsabrouty**

### CODE REVIEW

File: indicators.py [hunk 1]
```diff
@@ -8,8 +8,11 @@
 
 #   enriched_datamodel layer
     # generate_table_enriched_datamodel("emission_profile_ledger_enriched")
-    generate_table_enriched_datamodel("emission_profile_ledger_upstream_enriched")
+    # generate_table_enriched_datamodel("emission_profile_ledger_upstream_enriched")
```
Comment (SKruthoff) [inline indicators.py:11]: Comment back in

File: ecoInvent.py [hunk 2]
```diff
@@ -41,6 +41,8 @@
 
     generate_table_raw('inputProducts_raw')
     generate_table_raw('emissionData_raw')
+    # generate_table_raw('scope_2_mandatory_ghgs_raw')
```
Comment (SKruthoff) [inline ecoInvent.py:44]: Comment back in

**Review by SKruthoff — CHANGES_REQUESTED** [review]

**Review by drelsabrouty — COMMENTED** [review]
fixed

**Review by drelsabrouty — COMMENTED** [review]
fixed

**Review by drelsabrouty — COMMENTED** [review]
fixed

**Review by EuiYeonJang — APPROVED** [review]
It looks fine, I just don't know about leaving print statements

**Pull request merged**